sdk/storage/storage-blob/src/BlobDownloadResponse.ts

```diff
@@ -459,6 +459,17 @@ export class BlobDownloadResponse implements BlobDownloadResponseParsed {
     return this.originalResponse.versionId;
   }
 
+  /**
+   * Indicates whether version of this blob is a current version.
+   *
+   * @readonly
+   * @type {(boolean | undefined)}
```

Comment:
I don't remember whether we have a work item to track this, but we should switch from JSDoc to TSDoc syntax.  The tool should be able to figure out most of these information.

Reply:
Hmm, totally forgot about it. So many changes needed to migrate. Probably should write a script to do it.
I logged #13431 for storage and  will let go of this one now.

Reply:
I already have one open for each package as part of this epic: https://github.com/Azure/azure-sdk-for-js/issues/12959. Here is storage's: https://github.com/Azure/azure-sdk-for-js/issues/12956 .